### DISCUSSION TITLE
error-950 max number of cutbacks exceeded, terminating

### DISCUSSION THREAD

**Discussion by ghost** [damask-multiphysics/DAMASK#34] [Q&A] [answered]
I have oftenly faced the error as I presented in the title "max number of cutbacks exceeded, terminating". I guess that it is because of my severe deformation condition configured in rolling.yaml. file. I need to get deformation data when a material body is rolled significanly, i.e. over 70% reduction. Here I attach my rolling condition file, and I will very happy if an expert to modify the file for the severe reduction condition without facing error ! 

loadstep:
  - boundary_conditions:
      mechanical:
        dot_F: [[x, 0, 0],
                [0,  0, 0],
                [0,   0, -0.02]]
        P: [[0, x, x],
            [x, x, x],
            [x, x, x]]
    discretization:
      t: 10
      N: 40
    f_out: 4
  - boundary_conditions:
      mechanical:
        dot_F: [[x, 0, 0],
                [0,  0, 0],
                [0,   0, -0.04]]
        P: [[0, x, x],
            [x, x, x],
            [x, x, x]]
    discretization:
      t: 10
      N: 40
    f_out: 4

**Comment by hyharry** [comment]
you could increase the number of cut back, Pls have a look at the example numerics file

https://github.com/damask-multiphysics/DAMASK/blob/master/examples%2Fconfig%2Fnumerics.yaml

**Reply by eisenlohr** [reply, marked as answer]
Another possibility is to increase the number of increments used to discretize the deformation progression. Please see what happens when you, for instance, double the number of increments in the load steps.

Is the problem occurring during the first or second of your load steps? You are using different strain steps in each...
